src/Illuminate/View/ComponentAttributeBag.php

```diff
@@ -77,7 +108,7 @@ public function has($key)
      */
     public function missing($key)
     {
-        return ! $this->has($key, $this->attributes);
```

Comment:
`$this->attributes` has been removed from this function as it was and is not supported in the has function.